src/libstd/panicking.rs

```diff
@@ -159,6 +159,22 @@ pub fn take_hook() -> Box<Fn(&PanicInfo) + 'static + Sync + Send> {
 }
 
 /// A struct providing information about a panic.
+///
+/// You can get this object from the [`set_hook()`] function.
```

Comment:
Firstly this statement sounds like something that might get out of date quickly once we start adding more functions related to PanicInfo.

Secondly, I feel like phrasing of this could be improved. For example:

> A `PanicInfo` object is passed to a panic hook set by `set_hook()`

or

> `PanicInfo` is an argument to the panic hook (see `set_hook()`).